bubblepicker/src/main/java/com/igalata/bubblepicker/rendering/BubblePicker.kt

```diff
@@ -85,7 +85,7 @@ class BubblePicker : GLSurfaceView {
 
     private fun isClick(event: MotionEvent) = Math.abs(event.x - startX) < 20 && Math.abs(event.y - startY) < 20
 
-    private fun isTooFast() = System.currentTimeMillis() - previousUpTime < 250
+    private fun isTooFast() = System.currentTimeMillis() - previousUpTime <= 32
```

Comment:
Any reason for such a low value? Was it just from trial and error?

Reply:
Trial and error. But 32ms also represents 2 frames.